src/lcm/example_models_stochastic.py

```diff
@@ -62,13 +62,18 @@ def consumption_constraint(consumption, wealth):
 
 
 PARAMS = {
-    "beta": 0.25,
-    "utility": {"delta": 0.25, "gamma": 0.25},
-    "next_wealth": {"interest_rate": 0.25, "wage": 0.25},
+    "beta": 1.0,
+    "utility": {"delta": 0.5, "gamma": 0.25},
+    "next_wealth": {"interest_rate": 0.05, "wage": 10.0},
     "next_health": {},
     "consumption_constraint": {},
     "shocks": {
         "health": jnp.array([[[0.5, 0.5], [0.5, 0.5]], [[0.5, 0.5], [0.5, 0.5]]]),
-        "partner": jnp.array([[1.0, 0], [0.0, 1]]),
+        "partner": jnp.array(
+            [
+                [[[0, 1.0], [1.0, 0]], [[0, 1.0], [0.0, 1.0]]],
```

Comment:
Do we rely on these somewhere? If possible, numbers like `[[0.9, 0.1], [0.05, 0.95]]` or so would be even better (=more instructive for people using this as documentation). Same for health; `beta` could be 0.95 ?

Reply:
Very good point. We cannot change the degenerate distribution for the `partner` shock because the tests use this, however, I added more comments and made the distribution of `health` more realistic.